src/palette.st.css

```diff
@@ -45,6 +45,14 @@
   premiumMutedHover:	value(P30);
 }
 
+/* Input colors */
+:vars {
```

Comment:
I think that these variables should not sit in the palette but rather inside the component.
In the component - they should use vars from this palette file.

Regarding `inputHover`, I think Ben defined it with `mainHover` (B20) - Can you please contact him?
Regarding the `inputBackground` - use `backgroundSecondary`

Reply:
Since there are other "input" components (Like search etc) that use the same design, I thought it would be best to create variables that will be used by all inputs for consistency.

Regarding colors, I will talk to ben.

Reply:
@shlomitc `mainHover` and `backgroundSecondary` seem like very confusing names

Reply:
I mean it's absolutely not clear when I might want to use them.

Reply:
talked to Ben, the color is B50 for the input hover.